src/utils/api/helper.ts

```diff
@@ -398,13 +397,15 @@ export const retrieveCardData = async ({
   mixpanel.track(PAYMENT_PSPLIST_INIT.value, {
```

Comment:
PR looks great to me. But maybe in this point we could delete/comment this mixpanel invocation?

Reply:
We should eliminate all events in that method what do you think ?

Reply:
Just this event, because the psp list initialization start one api call down the stream, not precisely here in this point

Reply:
Fixed with 18445e92a68a78ef97f56923ba9781c1d3103e0b